nymea-app/ui/mainviews/WallboxView.qml

```diff
@@ -101,29 +101,29 @@ MainViewBase {
                             id: powerSwitch
                             checked: powerState.value
                             onClicked: {
-                                console.log(checked)
+                                root.setPower(powerState, checked)
                             }
                         }
 
                         Text {
+                            Layout.alignment: parent.horizontalCenter
                             anchors.horizontalCenter: parent.horizontalCenter
-                            text: Math.round(maxChargingSlider.value) + " " + qsTr("Ampere")
+                            text: Math.round(maxChargingSlider.value, 2) + " " + qsTr("Ampere")
                             color: "white"
                         }
 
                         Slider {
```

Comment:
Looks quite good. One thing though:

You want to use ThrottledSlider instead of the regular slider. If you work with slow devices, a normal slider would spam the system with those actions and the plugin might not be able to deal with so many requests. Because of that we have ThrottledSlider which waits for an action to be complete before sending the next one.

Search for TrottledSlider in the nymea:app codebase to find examples on how to use it.

Reply:
Done.